js/runtime.ts

```diff
@@ -39,26 +39,39 @@ window.onerror = (
   os.exit(1);
 };
 
-/*
-export function setup(mainJs: string, mainMap: string): void {
+// This is called during snapshot creation with the contents of
+// out/debug/gen/bundle/main.js.map.
+import { RawSourceMap } from "source-map";
+let mainSourceMap: RawSourceMap = null;
+function setMainSourceMap(rawSourceMap: RawSourceMap) {
+  util.assert(Number(rawSourceMap.version) === 3);
+  mainSourceMap = rawSourceMap;
+}
+window["setMainSourceMap"] = setMainSourceMap;
```

Comment:
You might want to reflect this in `lib.deno.d.ts` when that has landed.

Reply:
Thanks - done. (Although it's global right now. Will fix later)